src/paper_feeds/services/journal_service.py

```diff
@@ -0,0 +1,45 @@
+from paper_feeds.config import config
+from paper_feeds.db import get_engine
+from paper_feeds.models.journal import Journal, ISSN
+from paper_feeds.services import openalex
+from sqlmodel import Session, select
+
+
+def get_journals_homepage(journals: list[Journal]):
+    # extract one issn of each journal newly added to DB
```

Comment:
would be nice to move this into a docstring, eventually we will have docs!

Reply:
added docstrings to all methods in `openalex`, `journal_service` and `journal_repository`